<launch>
	<node pkg="djiros" type="djiros_node" name="djiros" output="screen">
		<env name="ROSCONSOLE_FORMAT" value="[${time}] ${message}"/>
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/ttyUSB.djiros"/>
		<param name="baud_rate" type="int" value="230400"/>
		<param name="app_id" type="int" value="$(env DJIROS_APPID)"/>
		<param name="app_api_level" type="int" value="2"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="12345678901234567890123456789012"/>
		<param name="enc_key" type="string" value="$(env DJIROS_ENCKEY)"/>	


	</node>
</launch>
